### DISCUSSION TITLE
Ready to Start Executing Tasks After Initial Setup - Seeking Guidance

### DISCUSSION THREAD

**Discussion by ayush-0110** [camicroscope/caMicroscope#729] [Q&A] [answered]
Hello @birm ,

I am Ayush Tiwari, and I have recently completed the initial setup for contributing to the caMicroscope project. I followed all the steps, including cloning the necessary repositories, installing Docker, and successfully running the caMicroscope application locally.
 I have reviewed the tasks mentioned in the project documentation and would like to confirm if it's the right time for me to begin working on these tasks. 

Could you please advise if I should start with the mentioned tasks or if there are any preliminary steps I should consider?

Thank you in advance for your guidance and support!

**Comment by ayush-0110** [comment]
Hi @birm . I am facing difficulty in getting the changes reflected on the localhost:4010 on my machine. I tried to make some changes on the landing page but can't get them reflected in the running application. Can you please help?

**Reply by birm** [reply, marked as answer]
Can you tell me a bit about how you set up your project to reflect changes?

The two ways I've typically done development are either mounting your caMicroscope cloned dir in Distro (see https://github.com/camicroscope/camicroscope?tab=readme-ov-file#fast-local-changes), or by pushing to a fork used within 'ca-back''s container. (first configure fork and branch under back, then after rebuilding back, you can run docker exec -it ca-back bash, then pull)

**Reply by gbengaoluwadahunsi** [reply]
@birm , thank you very much, this was what  I tried to explain on the other channel, you have clarified my ambiguity.

"The two ways I've typically done development are either mounting your caMicroscope cloned dir in Distro (see https://github.com/camicroscope/camicroscope?tab=readme-ov-file#fast-local-changes), or by pushing to a fork used within 'ca-back''s container. (first configure fork and branch under back, then after rebuilding back, you can run docker exec -it ca-back bash, then pull)"

**Reply by ayush-0110** [reply]
I created one directory and in that I have cloned the repositories.
Thanks for the help. I will try this out
